src/pages/billing/List.ts

```diff
@@ -0,0 +1,37 @@
+import { div, h } from 'react-hyperscript-helpers';
+import { ListPage } from 'src/billing/List/ListPage';
+import FooterWrapper from 'src/components/FooterWrapper';
+import TopBar from 'src/components/TopBar';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+
+interface ListProps {
+  queryParams: {
+    selectedName: string | undefined;
+  };
+}
+
+export const List = (props: ListProps) => {
+  const selectedName = props.queryParams.selectedName;
+  const breadcrumbs = 'Billing > Billing Project';
+
+  return h(FooterWrapper, { fixedHeight: true }, [
+    h(TopBar, { title: 'Billing', href: Nav.getLink('billing') }, [
+      !!selectedName &&
+        div({ style: Style.breadcrumb.breadcrumb }, [
+          div({ style: Style.noWrapEllipsis }, [breadcrumbs]),
+          div({ style: Style.breadcrumb.textUnderBreadcrumb }, [selectedName]),
+        ]),
+    ]),
+    h(ListPage, { queryParams: { selectedName } }),
```

Comment:
... and BillingList feels like a better name than ListPage, for this main sub-control.

Reply:
Done in 715a7363a4a302731ae1b4f719eede1ccdd846d4